config/x_ac_platform_compat.m4

```diff
@@ -45,16 +45,16 @@ AC_DEFUN([X_AC_SLURM_MPAO_WR], [
   AC_MSG_RESULT($enable_mpao_workaround)
 ])
 
-AC_DEFUN([X_AC_SHORT_HOSTNAME], [
+AC_DEFUN([X_AC_SGI_HOSTNAMES], [
```

Comment:
OK. I see this commit overwrite the earlier intermediate commit, ec20924b1042bfac1016e458f7f34e757ddfb879. Can you please `squash` the earlier intermediate commit? You should be able to accomplish this with `git rebase -i` technique like [here](https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history).